pages/linux/deluser.md

```diff
@@ -0,0 +1,20 @@
+# deluser
+
+> Delete user from system
+> All commands must be executed as root (`sudo [command]`)
+
+- Remove a user:
+
+`deluser {{name}}`
+
+- Remove a user and their home directory:
+
+`deluser --remove-home {{name}}`
+
+- Remove a user and their home, but backup it's files first:
```

Comment:
```suggestion
- Remove a user and their home, backing up their files:
```

Reply:
I didn't know how to put this well, but I would keep keep it as original (if it's not strongly against the rules)

Reply:
"it's" doesn't sound right&mdash;if you're referring to the user, I would use "their".

Reply:
Oh, also https://github.com/tldr-pages/tldr/pull/5245#discussion_r571437570

Reply:
How about this?
```suggestion
- Remove a user and their home, but backup their files into a `.tar.gz` file in the specified directory:
```

Reply:
I tried playing with it, and I discovered that pure `--backup` doesn't work and `--backup-to` needs to have exisitng directory (can't create a new one)

Should we add this?

Reply:
I'm not sure what you mean&mdash;my suggestion seems good enough.  If the directory is required to exist, well, I don't think we need to state that...many commands require the specified file/directory to exist (obviously :smile:)